test/powershell/Modules/Microsoft.PowerShell.Utility/MarkdownCmdlets.Tests.ps1

```diff
@@ -546,4 +546,25 @@ bool function()`n{`n}
             }
         }
     }
+
+    Context "Hosted PowerShell scenario" {
+        It 'ConvertFrom-Markdown gets expected out when run in hosted powershell' {
+            $pool = [runspacefactory]::CreateRunspacePool(1, 2, $Host)
+            $pool.Open()
+
+            $ps = [powershell]::Create()
+            $ps.RunspacePool = $pool
+            $ps.AddScript( {
+                    try {
+                        $output =  '# test' | ConvertFrom-Markdown
+                        $output.Html | Should -BeExactly '<h1 id="test">test</h1>'
+                    } catch {
+                        $_ | Out-Default
```

Comment:
I wonder why do we create this output if then send it to Out-Null?

Reply:
Removed